broadbean/sequence.py

```diff
@@ -1191,3 +1215,1236 @@ def rescaler(val, ampl, off):
                              jump_tos), self.channels)
 
         return output
+
+
+@deprecate(
+    reason="Class name signature should not be the same as Type Sequence",
+    alternative="PulseSequence",
+)
+class Sequence:
```

Comment:
simple aliasing should just work
```suggestion
Sequence = PulseSequence
```

or an empty subclass (well, it could have all the non-pep8-named methods that just call the pep8-named methods inside of them) will do the job and will avoid the need for copying code
```suggestion
@deprecate()
class Sequence(PulseSequence):
    pass
```